image_test.go

```diff
@@ -367,6 +368,55 @@ func TestImageRW(t *testing.T) {
 	}
 }
 
+func TestImageCubeRoundTrip(t *testing.T) {
+	strides := []int{2, 3, 6} // non repretitive, aperiodic strides
+
+	// the data looks like [0, 1, 2, ...]
+	data := make([]int16, 2*3*6)
+	for i := 0; i < len(data); i++ {
+		data[i] = int16(i - 32768)
+	}
+	var b bytes.Buffer
+	fits, err := Create(&b)
+	if err != nil {
+		t.Error(err)
+	}
+	im := NewImage(16, strides)
+	cards := []Card{
+		{Name: "BZERO", Value: 32768},
+		{Name: "BSCALE", Value: 1.}}
+	im.Header().Append(cards...)
+	err = im.Write(data)
+	if err != nil {
+		t.Error(err)
+	}
+	err = fits.Write(im)
+	if err != nil {
+		t.Error(err)
+	}
+	im.Close()
+	fits.Close()
+
+	// now read back
+	fits, err = Open(&b)
```

Comment:
there too (and `defer` close the returned FITS file handle as well)

Reply:
done